File: app/src/main/java/com/example/mad2/pageone/PageOneVM.java
package com.example.mad2.pageone;

import android.os.Bundle;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class PageOneVM extends ViewModel {
    public MutableLiveData<PageOneModel> pageOneModel = new MutableLiveData<>(new PageOneModel());
    private MutableLiveData<Bundle> navArguments = new MutableLiveData<>();

    public MutableLiveData<PageOneModel> getPageOneModel() {
        return pageOneModel;
    }

    public MutableLiveData<Bundle> getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments.setValue(navArguments);
    }

}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/pageone/data/viewmodel/PageOneVM.kt
package com.jiahuisapplication.app.modules.pageone.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.pageone.`data`.model.PageOneModel
import org.koin.core.KoinComponent

class PageOneVM : ViewModel(), KoinComponent {
  val pageOneModel: MutableLiveData<PageOneModel> = MutableLiveData(PageOneModel())

  var navArguments: Bundle? = null
}
